<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Beautifier::Internal::Uncrustify::UncrustifyOptionsPage</class>
 <widget class="QWidget" name="Beautifier::Internal::Uncrustify::UncrustifyOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
   <item>
    <widget class="QGroupBox" name="configuration">
     <property name="title">
      <string>Configuration</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="commandLabel">
        <property name="text">
         <string>Uncrustify command:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="Utils::PathChooser" name="command" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="mimeLabel">
        <property name="text">
         <string>Restrict to MIME types:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="mime"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="options">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="useOtherFiles">
        <property name="text">
         <string>Use file uncrustify.cfg defined in project files</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="useSpecificFile">
          <property name="text">
           <string>Use file specific uncrustify.cfg</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Utils::PathChooser" name="uncrusifyFilePath" native="true"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="useHomeFile">
        <property name="text">
         <string extracomment="HOME is replaced by the user's home directory">Use file uncrustify.cfg in HOME</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,9999">
        <item>
         <widget class="QCheckBox" name="useCustomStyle">
          <property name="text">
           <string>Use customized style:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Beautifier::Internal::ConfigurationPanel" name="configurations" native="true"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="formatEntireFileFallback">
        <property name="toolTip">
         <string>For action Format Selected Text</string>
        </property>
        <property name="text">
         <string>Format entire file if no text was selected</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Beautifier::Internal::ConfigurationPanel</class>
   <extends>QWidget</extends>
   <header location="global">beautifier/configurationpanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
